code/modules/gear_presets/synths.dm

```diff
@@ -491,6 +491,59 @@
 	H.equip_to_slot_or_del(new /obj/item/device/lightreplacer(H.back), WEAR_IN_R_STORE)
 	H.equip_to_slot_or_del(new /obj/item/maintenance_jack(H), WEAR_J_STORE)
 
+
+/datum/equipment_preset/synth/working_joe/engi
+	name = "Synthetic - Hazmat Joe"
+	flags = EQUIPMENT_PRESET_START_OF_ROUND|EQUIPMENT_PRESET_MARINE
+	faction = FACTION_MARINE
+	faction_group = list(FACTION_MARINE)
+	assignment = JOB_WORKING_JOE
+	rank = JOB_WORKING_JOE
+	skills = /datum/skills/working_joe
+	languages = list(LANGUAGE_ENGLISH, LANGUAGE_APOLLO, LANGUAGE_RUSSIAN, LANGUAGE_JAPANESE, LANGUAGE_GERMAN, LANGUAGE_SPANISH, LANGUAGE_CHINESE)
+
+/datum/equipment_preset/synth/working_joe/New()
+	. = ..()
+	access = get_all_accesses()
```

Comment:
you're calling parent here, any /datum/equipment_preset/synth/New() already gets all access, so this is redundant